discovery-provider/src/tracer.py

```diff
@@ -0,0 +1,14 @@
+# mypy: ignore-errors
+from opentelemetry import trace
+from opentelemetry.instrumentation.logging import LoggingInstrumentor
+from opentelemetry.instrumentation.requests import RequestsInstrumentor
+from opentelemetry.sdk.resources import SERVICE_NAME, Resource
+from opentelemetry.sdk.trace import TracerProvider
+
+
+def configure_tracer():
+    trace.set_tracer_provider(
+        TracerProvider(resource=Resource.create({SERVICE_NAME: "discovery-provider"})),
+    )
+    LoggingInstrumentor().instrument()
```

Comment:
Same question about if the tracer ID is exposed in both Loggly and ELK.

Reply:
Yes